visualize_packwerk.gemspec

```diff
@@ -24,12 +24,12 @@ Gem::Specification.new do |spec|
   spec.required_ruby_version = '>= 2.6'
 
   spec.add_dependency 'sorbet-runtime'
+  spec.add_dependency 'packs'
   spec.add_dependency 'parse_packwerk'
   spec.add_dependency 'code_ownership'
   spec.add_dependency 'rake'
   spec.add_dependency 'ruby-graphviz'
 
-  spec.add_development_dependency 'bundler', '~> 2.2.16'
```

Comment:
I think we'd still want this if we choose to continue to commit the `Gemfile.lock`.